packages/upset/src/components/Sidebar.tsx

```diff
@@ -97,33 +106,88 @@ export const Sidebar = () => {
             <RadioGroup
               value={sortBy}
               onChange={(ev) => {
-                actions.sortBy(ev.target.value as SortBy);
+                actions.sortBy(ev.target.value as SortBy, 'Ascending');
               }}
             >
-              {sortByList.map((sort) => (sort === 'Deviation' ?
-                (
-                  <Alert severity="info" variant="outlined" role="generic" key={sort} sx={{ alignItems: 'center', padding: '0.1em 0.4em', marginTop: '0.5em' }}><Typography>Use column headers for custom sorting</Typography></Alert>
-                ) :
-                (
-                  <Box
-                    css={itemDivCSS}
-                    key={sort}
-                    aria-label={helpText.sorting[sort]}
-                    onKeyDown={(e) => {
-                      if (e.key === 'Enter') {
-                        actions.sortBy(sort);
+              <Box
+                css={itemDivCSS}
+                key="Degree"
+                aria-label={helpText.sorting['Degree']}
+                onKeyDown={(e) => {
+                  if (e.key === 'Enter') {
+                    actions.sortBy('Degree', 'Ascending');
+                  }
+                }}
+              >
+                <FormControlLabel
+                  key="Degree"
+                  value="Degree"
+                  label={
+                    <Box
+                      css={itemDivCSS}
+                      onClick={() => {
+                        if (sortBy === 'Degree') {
+                          if (sortByOrder === 'Ascending') {
+                            actions.sortBy('Degree', 'Descending');
+                          } else {
+                            actions.sortBy('Degree', 'Ascending');
+                          }
+                        }
+                      }}
+                    >
+                      <Typography>Degree</Typography>
+                      { sortBy === 'Degree' &&
+                        (sortByOrder === 'Ascending' ?
+                          <ArrowUpward sx={arrowIconCSS} /> :
+                          <ArrowUpward sx={{ ...arrowIconCSS, transform: 'rotate(180deg)' }} />
+                        )
                       }
-                    }}
-                  >
-                    <FormControlLabel
-                      key={sort}
-                      value={sort}
-                      label={sort}
-                      control={<Radio size="small" />}
-                    />
-                    <HelpCircle text={helpText.sorting[sort]} />
-                  </Box>
-                )))}
+                    </Box>
```

Comment:
How is this for accessibility? Would a blind person be able to tell the sort direction from this html?

Reply:
Same for the size sort